test/api/stakeholdergroup/api_test.go

```diff
@@ -0,0 +1,86 @@
+package stakeholdergroup
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/client"
+)
+
+func TestStakeholderGroupCRUD(t *testing.T) {
```

Comment:
@jortel Would this way of naming variables etc in this test have better fit than more verbose (more different variable names depending on tested resource type) version started in application test?